cli-feed-v3-3.json

```diff
@@ -0,0 +1,239 @@
+{
+  "tags": {
+    "v1": {
+      "release": "1.13.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    },
+    "v1-prerelease": {
+      "release": "1.13.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v2": {
+      "release": "2.60.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    },
+    "v2-prerelease": {
+      "release": "2.60.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v2-preview": {
+      "release": "2.60.0",
+      "releaseQuality": "Preview",
+      "hidden": true
+    },
+    "v3-preview": {
+      "release": "3.20.0",
+      "releaseQuality": "Preview",
+      "hidden": true
+    },
+    "v3-prerelease": {
+      "release": "3.20.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v3": {
+      "release": "3.20.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    }
+  },
+  "releases": {
+    "1.13.0": {
+      "cli": "https://functionscdn.azureedge.net/public/1.0.28/Azure.Functions.Cli.zip",
+      "localEntryPoint": "func.exe",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/1.0.3.10182.zip",
+      "sha2": "BB4978D83CFBABAE67D4D720FEC1F1171BE0406B2147EF3FECA476C19ADD9920",
+      "runtimeInformation": {
+        "dotnet": {
+          "netframework-inproc": {
+            "displayInfo": {
+              "displayName": ".NET Framework",
+              "hidden": false,
+              "diplayVersion": "v1",
+              "targetFramework": ".NET Framework"
+            },
+            "capabilites": "inProc",
+            "sdkName": "Microsoft.NET.Sdk.Functions",
+            "sdkVersion": "1.0.29" ,
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/1.0.3.10324",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/1.0.3.10324",
+            "projectTemplateId": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.1.x",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~1"
+            },
```

Comment:
Let's add          "FUNCTIONS_WORKER_RUNTIME": null here, so that if someone (not sure why) were to publish a v1 project to a function app that was on v2, we would clear the property.  Scenario-wise it's not super necessary, but testing-wise it proves we handle clearing properties.

Reply:
@phile-allen-msft, I have added `FUNCTIONS_WORKER_RUNTIME: dotnet` as technically that's a valid setting and is usually just assumed if not there for v1 I think.